Adds `REAMDE.md` template for all new packages (#1)
pulsar-cooperative/.github: package-template/.github/issue-comment-template.md

# Welcome to Pulsar Cooperative: {{ .repo }}!

> Thanks a ton [{{ .user }}]({{ .userLink }}) for creating this issue!
> 
> We can't understate how important, and appreciated it is to contribute!

Reviewer: ```suggestion
Thanks a ton [{{ .user }}]({{ .userLink }}) for creating this issue! We can't understate how important and appreciated it is for your contribution!
```

> But it's important to be clear about the level of support that may be expected here.

Reviewer: ```suggestion
However, it's important to be clear about the level of support that should be expected here.
```

> As this repository is within the Pulsar Cooperative Organization, you cannot expect any core members of the Pulsar Organization to offer advice, or code fixes. The fix to your issue is expected to come from someone else in the community just like yourself, as is the mission of this organization. If you'd like, feel free to read more about this in our organization's [Contributing Guide](https://github.com/pulsar-cooperative/.github/blob/main/CONTRIBUTING.md).

Reviewer: ```suggestion
As this repository is within the [Pulsar Cooperative Organization](https://github.com/pulsar-cooperative), you cannot expect any core members of the Pulsar Organization to offer advice or code fixes. The fix to your issue is expected to come from someone else in the community just like yourself, as is the mission of this organization. If you'd like, feel free to read more about this in our organization's [Contributing Guide](https://github.com/pulsar-cooperative/.github/blob/main/CONTRIBUTING.md).
```
